### PR TITLE
horbyshyn react-basics homework

### PR DESCRIPTION
**Pull request by horbyshyn**

### CODE REVIEW

**Comment by KinjinZzz** [comment]
@horbyshyn there must be no submodules, please remove it and add homework directly to your folder.

**Comment by horbyshyn** [comment]
> @horbyshyn there must be no submodules, please remove it and add homework directly to your folder.

fixed

File: homeworks/react-basics/horbyshyn/src/App.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+class LecturesData extends React.Component{
+  state = {
+    lectures: []
+  }
+  componentDidMount() {
+    axios.get('https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2019/master/homeworks/react-basics/data.json')
+      .then(res => {
+        const lectures = Object.values(res.data.lectures);
+        this.setState({ lectures });
+      })
```
Comment (KinjinZzz) [inline homeworks/react-basics/horbyshyn/src/App.js]: Don't forget to `catch()` errors

File: homeworks/react-basics/horbyshyn/src/App.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      <div className="lectures">
+      {this.state.lectures.map((item, i) => (
+          RenderLecture(item, i)
+      ))}
+      </div>
+    );
+  }
+}
+
+function RenderLecture(item, key){
```
Comment (KinjinZzz) [inline homeworks/react-basics/horbyshyn/src/App.js]: It is bad practice to have few components in single file

File: homeworks/react-basics/horbyshyn/src/App.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  function handleClick(index) {
+    console.log(index);
+    var id = "lecture-"+index;
+    console.log(document.getElementById(id).classList);
+    if(document.getElementById(id).classList.contains('hidden'))
```
Comment (KinjinZzz) [inline homeworks/react-basics/horbyshyn/src/App.js]: Transform `Render Lecture` to class component and use state `hidden`, then you can handle state `onClick()` and render Lecture info only if `hidden` is `false`. You have to control logic in react using state, using dom will cause problem with rerender and will cause conflicts with few components with the same indexes.

**Review by Kinjinka — CHANGES_REQUESTED** [review]

**Comment by denysdovhan** [comment]
Closed since requested changes aren't applied.

**Pull request closed without merging**